test/functional/test_runner.py

```diff
@@ -344,7 +344,7 @@ def main():
     parser.add_argument('--keepcache', '-k', action='store_true', help='the default behavior is to flush the cache directory on startup. --keepcache retains the cache from the previous testrun.')
     parser.add_argument('--quiet', '-q', action='store_true', help='only print dots, results summary and failure logs')
     parser.add_argument('--tmpdirprefix', '-t', default=tempfile.gettempdir(), help="Root directory for datadirs")
-    parser.add_argument('--failfast', action='store_true', help='stop execution after the first test failure')
+    parser.add_argument('--failfast', '-F', action='store_true', help='stop execution after the first test failure')
```

Comment:
Any reason this isn't lowercase, like all the other options?

Reply:
Lowercase `-f` would typically be reserved for "force", though... :/

Reply:
I'm seeing [test/functional/wallet_hd.py help (in which the lowercase f is being used as short option)][0] being automatically printed after requesting `test/functional/test_runner.py -h`.

Using an uppercase F here has been a precaution, but in this occasion I would be keen to switch it to lowercase here and deal with a potential arguments conflict later, once and if I'm sure it can happen.

---

BTW, it would appeal me to change this lowercase f in test_framework.py to an uppercase D (reserving lowercase d for --descriptors):
https://github.com/bitcoin/bitcoin/blob/446b706696451ae1a66ac416f347d734c5741d7c/test/functional/test_framework/test_framework.py#L203


[0]: https://github.com/bitcoin/bitcoin/blob/446b706696451ae1a66ac416f347d734c5741d7c/test/functional/test_framework/test_framework.py#L203

Reply:
Note https://stackoverflow.com/a/48798075.

Reply:
CC @josibake

Reply:
I would see useful using an "s" for "stop" instead.